test/parallel/test-cluster-disconnect.js

```diff
@@ -1,41 +1,41 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
 
 if (cluster.isWorker) {
-  net.createServer(function(socket) {
+  net.createServer((socket) => {
     socket.end('echo');
   }).listen(common.PORT, '127.0.0.1');
 
-  net.createServer(function(socket) {
+  net.createServer((socket) => {
     socket.end('echo');
   }).listen(common.PORT + 1, '127.0.0.1');
 
 } else if (cluster.isMaster) {
   var servers = 2;
 
   // test a single TCP server
-  var testConnection = function(port, cb) {
-    var socket = net.connect(port, '127.0.0.1', function() {
+  const testConnection = function(port, cb) {
+    var socket = net.connect(port, '127.0.0.1', () => {
       // buffer result
       var result = '';
-      socket.on('data', function(chunk) { result += chunk; });
+      socket.on('data', common.mustCall((chunk) => { result += chunk; }));
```

Comment:
Remote chance yes, but in every run of this test that I've seen, it's only happened once. If it starts happening more than once, then there's a chance that something internal has changed that may not be correct. This would highlight such a change, and if it's legitimate, then simply updating the mustCall count should be sufficient.